crates/relayer/src/chain/ckb4ibc.rs

```diff
@@ -752,10 +778,12 @@ impl ChainEndpoint for Ckb4IbcChain {
         _request: QueryConsensusStateRequest,
         _include_proof: IncludeProof,
     ) -> Result<(AnyConsensusState, Option<MerkleProof>), Error> {
+        let status = self.query_application_status()?;
         Ok((
             AnyConsensusState::Ckb(CkbConsensusState {
-                timestamp: Time::now(),
-                commitment_root: CommitmentRoot::from_bytes(&[]),
+                timestamp: status.timestamp.into_tm_time().expect("timestamp to time"),
+                // no commitment root for Ckb chain
+                commitment_root: CommitmentRoot::from(vec![]),
```

Comment:
Should we use live cell commitment after the CKB2023 hard fork? If so, please add a comment here.

Reply:
I'm not sure this field can be used as live cell commitment root, I should check its logic